src/gtfs/mod.rs

```diff
@@ -504,34 +504,10 @@ struct Route {
     sort_order: Option<u32>,
 }
 
-fn remove_stop_zones(model: Model) -> Collections {
-    let mut collections = model.into_collections();
-    collections
-        .stop_points
-        .retain(|sp| sp.stop_type != StopType::Zone);
-
-    let stop_point_ids: Vec<Idx<StopPoint>> = collections
-        .stop_points
-        .get_id_to_idx()
-        .values()
-        .copied()
-        .collect();
-
-    collections.vehicle_journeys.retain(|vj| {
-        vj.stop_times
-            .iter()
-            .all(|st| stop_point_ids.contains(&st.stop_point_idx))
-    });
-
-    collections
-}
-
 /// Exports a `Model` to [GTFS](https://gtfs.org/reference/static) files
 /// in the given directory.
 /// see [NTFS to GTFS conversion](https://github.com/CanalTP/transit_model/blob/master/src/documentation/ntfs2gtfs.md)
 pub fn write<P: AsRef<Path>>(model: Model, path: P) -> Result<()> {
-    let collections = remove_stop_zones(model);
```

Comment:
I think I understand why you did move this line directly to the connector, but it seems it would be better to keep it here, so anything that is writing GTFS will be impacted. With your solution, we would need to think about adding a call to `remove_stop_zones` on every new binary producing some GTFS.

Reply:
OK, so we discussed it together and the conclusion is: yes, it is true that if we ever have another binary that is writing GTFS, we need to not forget to add a call `remove_stop_zones()`. However:
- it is still very unlikely we have to create a new binary that write GTFS (we usually convert to or from NTFS)
- even in the case of such a binary, it is very unlikely that we will end up with `pickup_type` and/or `drop_off_type` with a value of 3
- this PR probably improved performance by an order of magnitude by avoiding transforming a `Model` back to a `Collections`, then back again to a `Model` (which is the main reason why the call to `remove_stop_zones()` has been moved in the first place).

Therefore, we're ok to lose a bit of a "_maybe more logical_" place to call `remove_stop_zones()` in the profit of performance.